crypto/err/openssl.txt

```diff
@@ -1754,7 +1754,7 @@ X509_R_AKID_MISMATCH:110:akid mismatch
 X509_R_BAD_SELECTOR:133:bad selector
 X509_R_BAD_X509_FILETYPE:100:bad x509 filetype
 X509_R_BASE64_DECODE_ERROR:118:base64 decode error
-X509_R_CANT_CHECK_DH_KEY:114:cant check dh key
+X509_R_CANT_CHECK_DH_KEY:114:can't check dh key
```

Comment:
Auto-generated

Reply:
Reverted...

Reply:
If you have reverted it is not committed.

Reply:
Ahh I see further down there are comments